### ISSUE TITLE
Remove timezone - no summer time / browser altertions

### ISSUE THREAD

**Issue by rmorgan3128** [nhn/tui.calendar#671]
Hey there,

I'm currently creating a webapp with the tui vue calendar library. The app is going to be used by multiple people from different countries to create and set event "reservation" times. 

I've created a custom popup which allows the user to set the start and end time of their reservation. This is great and the monthly view works well.

Weekly and Daily views also look good on the initial surface however I've noticed that the reservations are getting affected by what I believe is my browser/system settings? For example the events currently show starting at 9AM however when BST finishes in October this year the starting time moves to 8AM.

Is there anyway to remove this? I want a flat calendar really. A reservation/event will always take place on the same day regardless of the timezone. 

The whole timezone confuses me a bit to be honest though. Maybe its just my use case but I can't figure why you'd want an event to change time by default?

I tried setting:

` timezones: [{
        timezoneOffset: 0,
        displayLabel: 'UTC',
        tooltip: 'UTC'
      }],`

But the same seems to happen in October when BST time starts. 

This also causes a bug with early morning reservations. For example events starting at 00:00:00 -> 04:00:00 gets moved to the previous day causing the bar to be spread over 2 days.

I had thought about implementing something that would alter the time clientside for the time being (add an hour in October) so it appears correct in the weekly / daily view but I'm a bit concerned this is going to be different for all the countries that login to the app?

https://fullcalendar.io/docs/timeZone seems to have a "UTC" mode but I was hoping to not have to move over to that library as I've really enjoyed tui calendar. 

TLDR: I just want the event to always start and end at the same time. Regardless of timezone / summer time changes etc.

Any suggestions or even hacks for the time being would be appreciated :)

**Comment by dilyanpalauzov** [comment]
Does calling
```javascript
new Calendar(whatever, {timezones:[{timezoneOffset:0}])
```
help to ignore the timezones of the browser?

**Comment by rmorgan3128** [comment]
Hey @dilyanpalauzov,

Thanks for the reply. 

It does help but there's still a few issues or maybe something I'm missing.

So for example I have the datetime stamp - 2020-07-27 09:00:00

If I apply the the `timezoneOffset:0` then it appears on the weekly/daily view at 8AM. This doesn't change though and is the same in October when BST ends. So this makes me feel like the dateTime stamp is still getting calculated from my UK timezone?

If I keep `timezoneOffset:0` but apply Z to the end of my datetime stamp (for UTC I believe) then the event appears correctly at 9AM however when BST ends in October the start time goes back to 8AM. 

Not sure if I'm missing something here? All I want is for the event to stay the same no matter what.

**Comment by dilyanpalauzov** [comment]
I do not umderstand it neither.  Does is appear correctly the other views: one week, one day.

**Comment by rmorgan3128** [comment]
![utc-timezone](https://user-images.githubusercontent.com/68780175/88530403-3cad0600-cff9-11ea-9438-876b796e5261.JPG)
![no-utc-time](https://user-images.githubusercontent.com/68780175/88530433-47679b00-cff9-11ea-94a9-74b29625cb2d.JPG)

Hey @dilyanpalauzov,

So those are the two views for the different options from above.

Both are set as `timezoneOffset:0` however the first has been passed over as a UTC datetime stamp and the second is just a plain one.

So 2020-07-09T09:00:00Z vs 2020-07-09 09:00:00 

My worry is that several different countries will login to the app and see different times on those views and I dont want to concern myself with timezones.

Many thanks

**Comment by rmorgan3128** [comment]
Just had a further test by changing my system settings @dilyanpalauzov.

Passing 2020-07-09T09:00:00Z seems to be consistent apart from around October time when BST/DST changes. 

For example I tried UTC+0 = 24th October 9AM -> 25th October 8AM
                               UTC+1 = same as above
                               UTC+3 = No DST time in 2020 so stays at 9AM all year
                               UTC+12 = 26th October 9AM, 27th October 10AM

By not using the UTC timestamp the start time changes completely for each day. 

Is there a way to disable DST/BST I wonder? I guess I could always manipulate the times around those dates but I wouldn't want to maintain that long term.

**Comment by dilyanpalauzov** [comment]
I agree.  I set options={timezones:[{timezoneOffset:}]} tgen I add an event that runs every day from 18:30-19:30.  Once DTS passes by srolling the valendar, the event suddently moves to 17:30-18:30.  But in spring, when DST relly changes I guess those events from 17:30-18:30 will move in the same browser to 18:30-19:30.

TUI calendar shall really have an option not to do automatic timezone conversions.

**Comment by jungeun-cho** [comment]
@rmorgan3128 
As a DST-related issue, it is a calendar rendering issue.

**Comment by luismabenitez** [comment]
Any update on this issue?

**Comment by dilyanpalauzov** [comment]
I personally deleted the TZDate class, use instead Date, always pass time to the constructor of Date and it works for me.  Except on the switch to/from DST, where one day events last for 25/23 hours, but this is very minor thing.

**Comment by jungeun-cho** [comment]
I think we should solve this problem as soon as possible.
I'm hoping it will be resolved next week.

**Comment by Nitrodist** [comment]
I don't understand how the commit above solves the issue of tui.calendar always setting the calendar to reflect the user's timezone that the browser is sending along @jungeun-cho 

I would like an option to present the timezone explicitly of the entire Calendar instance. 

My use case is that I have a business whose primary business location is in a specific city (i.e., timezone). When users log in from a different timezone, I want to always see the calendar display times based on the primary business location's time zone and not the user's browser.
